docs/sphinx/about/whats-new.txt

```diff
@@ -1,6 +1,45 @@
 Version history
 ===============
 
+5.1.9 (2016 April 11)
+---------------------
+
+* Java bug fixes, including:
+   - SDT
+       - fixed width padding calculation for single-pixel image.
+   - Deltavision
+       - fixed the parsing of the new date format.
+       - added support for parsing and storing the working distance in native units.
+   - Micromanager
+       - cleaned up JSON metadata parsing.
+   - Olympus Fluoview
+       - fixed null pointer exceptions while parsing metadata.
+   - Leica LIF
+       - fixed large milt-tile files from having incorrect plane offsets after the 2GB mark.
+   - EM formats
```

Comment:
Clarify that this is MRC and Spider specifically (per https://github.com/openmicroscopy/bioformats/pull/2298/files)?